src/main/kotlin/parts/wisdom/utils/RandomLineGenerator.kt

```diff
@@ -0,0 +1,52 @@
+package parts.wisdom.utils
+
+import me.joypri.to
+import parts.wisdom.arcraider.Color
+import parts.wisdom.arcraider.Direction
+import parts.wisdom.arcraider.Length
+import parts.wisdom.arcraider.TheColor
+import parts.wisdom.arcraider.TheDirection
+import parts.wisdom.arcraider.X
+import parts.wisdom.arcraider.Y
+import parts.wisdom.hdezninirola.lineworld.Line
+import java.util.*
+
+
+class RandomLineGenerator(seed: Long? = null, gridWidth: Int, gridHeight: Int) {
+
+    private val rnd = seed?.let { Random(it) } ?: Random()
+
+    val lineGeneratorSequence = sequence {
+        while (true) {
+            val direction = Direction.values()[(0..1).random()]
```

Comment:
The constant range `0..1` is of course fragile. Perhaps `Direction.values().shuffled().take(1)[0]`?

Reply:
`Direction.values().indices.random()` is perfect for this